README.md

```diff
@@ -2,29 +2,46 @@
 # AAVE Starknet Bridge
 
 [![Tests](https://github.com/aave-starknet-project/aave-starknet-bridge/actions/workflows/ci.yml/badge.svg)](https://github.com/aave-starknet-project/aave-starknet-bridge/actions/workflows/ci.yml)
-[![GitHub pull requests](https://img.shields.io/github/issues-pr/cdnjs/cdnjs.svg?style=flat)](https://github.com/aave-starknet-project/aave-starknet-bridge/pulls)
-[![Issues](https://img.shields.io/github/issues-raw/tterb/PlayMusic.svg?maxAge=25000)](https://github.com/aave-starknet-project/aave-starknet-bridge/issues)
-
- <a href="https://github.com/aave-starknet-project/aave-starknet-bridge/blob/main/LICENSE.md"><img src="https://img.shields.io/github/license/sourcerer-io/hall-of-fame.svg?colorB=ff0000"></a>
+[![Tests](https://github.com/aave-starknet-project/aave-starknet-bridge/actions/workflows/deploy.yml/badge.svg)](https://github.com/aave-starknet-project/aave-starknet-bridge/actions/workflows/deploy.yml)
+<a href="https://github.com/aave-starknet-project/aave-starknet-bridge/blob/main/LICENSE.md"><img src="https://img.shields.io/github/license/sourcerer-io/hall-of-fame.svg?colorB=ff0000"></a>
 
 :warning: This codebase is still in an experimental phase, has not been audited, might contain bugs and should not be used in production.
 
+
+## Table of contents
+  * [Overview](#overview)
+  * [Architecture](#architecture)
+  * [Contracts](#contracts)
+  * [ETHStaticATokens on L2](#ethstaticatokens-on-l2)
+  * [Bridging aTokens from L1<>L2](#bridging-atokens-from-l1--l2)
+    + [Transfer L1->L2](#transfer-l1--l2-)
+    + [Transfer L2->L1](#transfer-l2--l1-)
+  * [Synchronisation of rewards on L1 <> L2](#synchronisation-of-rewards-on-l1----l2)
+  * [Claiming rewards on L2](#claiming-rewards-on-l2)
+  * [Bridging rewards from L2->L1](#bridging-rewards-from-l2--l1)
+  * [Proxies](#proxies)
+  * [Governance](#governance)
+  * [Development Setup](#environment)
+  * [Tests](#run-the-tests)
+  * [Deployment](#deployment)
+
+
+
 ## Overview
 
 For Aave, one of the main current and future goals is growth of liquidity and user base. As seen on side-chains with low transaction cost like Polygon or Avalanche, there is high demand to use the Aave protocol with small amounts to earn high yield. That's why we brought to you an initial phase of the Aave <> Starknet integration allowing deposit/withdrawal on Aave Ethereum by exclusively transacting on Starknet.
 
-The bridge allows users to deposit and withdraw `staticATokens` - wrappers converting balance-increasing [aTokens]( https://docs.aave.com/developers/tokens/atoken) into exchange-rate-increasing staticATokens - on StarkNet and get wrapped tokens `ETHStaticATokens` that allow users to keep enjoying the same rewards as on L1. 
+The bridge allows users to deposit and withdraw their `aTokens` on StarkNet and get wrapped tokens `ETHStaticATokens` that allow users to keep enjoying the same rewards as on L1. 
```

Comment:
Why removing the link to `aToken definition? Also, you should add a small description for staticAToken. Readers coming to this repo are not all knowledgeable about Aave infra.

Reply:
restored :)